website/content/_index.md

```diff
@@ -12,24 +12,28 @@ hero_section:
     link: https://prql-lang.org/book/
     label: "Reference"
   prql_example: |
-    from employees
-    filter start_date > @2021-01-01
```

Comment:
WDYT about keeping a date one? I think that's a nice PRQL feature

Reply:
Yup, good point.